module-2/backend/week_two.md

```diff
@@ -1,10 +1,11 @@
 ## Week Two - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+active record is our Object relational mapper. it allows us to work with MYsql DBs via Ruby
```

Comment:
it would be sql databases in general. MySQL is a proprietary db from oracle but yes we can use active record with it.